README.md

```diff
@@ -87,13 +87,13 @@ The next two sections presume you are using the out-of-the-box `.mf.config.json`
 
 # Firebolt SDK Support
 
-Mock Firebolt supports these Firebolt SDKs: **core**, **manage**, **discovery**.
+Mock Firebolt supports these Firebolt SDKs: **core**, **discovery**.
 
-By default, only the core SDK is enabled, meaning if you try to make calls to methods in the other SDKs, Mock Firebolt will thow an error.
+By default, only the core sdk retrieved from HTTP is enabled
```

Comment:
Again, I don't think this is quite right... Only the unified OpenRPC retrieved via HTTP is used, but this does mean that both/all SDKs are 'enabled' (supported).

Reply:
Agreed. We can say "only the core/manage SDK" or similar.

Reply:
Addressed